### PR TITLE
Update Splashscreen Plugin Doc for Cordova 9.x

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected
none

### Motivation and Context
Saw doc changes but not reflected

### Description
Created Cordova 9.x Snapshot

### Testing
none

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

**Comment by janpio** [comment]
Those "pull in docs" things only happen in the version directly, not dev?
What are the commands to do such a targeted pull in of one plugin doc?

**Comment by erisu** [comment]
> Those "pull in docs" things only happen in the version directly, not dev?

Your changes is already in dev: https://cordova.apache.org/docs/en/dev/reference/cordova-plugin-splashscreen/index.html

> What are the commands to do such a targeted pull in of one plugin doc?
As far as I know, there is no command to pull in one thing or specific thing. There is only a single snapshot command that will pull everything from `dev` into the last released version `9.x` => `npm run update-docs`.

**Comment by janpio** [comment]
> > Those "pull in docs" things only happen in the version directly, not dev?
> 
> Your changes is already in dev: https://cordova.apache.org/docs/en/dev/reference/cordova-plugin-splashscreen/index.html

Oh, how did that happen...

**Comment by erisu** [comment]
Wild guess: You made these changes x days ago and your effort to get SVN credentials working to release iOS deployed it because its part of the build process.

**Comment by erisu** [comment]
@janpio is this still on hold? I think we already released all plugins so maybe we can go ahead and merge and close out this PR.

File: www/docs/en/9.x/reference/cordova-plugin-splashscreen/index.md [hunk 1]
```diff
@@ -47,12 +64,96 @@ This plugin is required to work with splash screens. This plugin displays and hi
...
+    <splash src="res/screen/ios/Default~iphone.png" width="320" height="480"/>
+    <splash src="res/screen/ios/Default@2x~iphone.png" width="640" height="960"/>
+    <splash src="res/screen/ios/Default-Portrait~ipad.png" width="768" height="1024"/>
+    <splash src="res/screen/ios/Default-Portrait@2x~ipad.png" width="1536" height="2048"/>
+    <splash src="res/screen/ios/Default-Landscape~ipad.png" width="1024" height="768"/>
+    <splash src="res/screen/ios/Default-Landscape@2x~ipad.png" width="2048" height="1536"/>
+    <splash src="res/screen/ios/Default-568h@2x~iphone.png" width="640" height="1136"/>
+    <splash src="res/screen/ios/Default-667h.png" width="750" height="1334"/>
+    <splash src="res/screen/ios/Default-736h.png" width="1242" height="2208"/>
+    <splash src="res/screen/ios/Default-Landscape-736h.png" width="2208" height="1242"/>
```
Comment (breautek) [inline www/docs/en/9.x/reference/cordova-plugin-splashscreen/index.md:126]: The legacy launch images can probably be removed now as they are no longer supported by Apple, right?

Comment (erisu) [reply www/docs/en/9.x/reference/cordova-plugin-splashscreen/index.md:126]: This is Cordova 9 docs which should still reference this.
Dev & Cordova 10 should have it removed remove it when pulled.

**Pull request merged**